### DISCUSSION TITLE
Image previewer bugged on ST

### DISCUSSION THREAD

**Discussion by giuskywalker** [sxyazi/yazi#609] [1. Q&A] [answered]
Hello guys, just installed yazi on my archlinux machine, i'm using ST with some patches, for some reason my image preview looks kinda strange, anybody had this issue before?
![2024-02-01_07-39-24](https://github.com/sxyazi/yazi/assets/83036320/53e14824-d3d5-498e-876e-b48430f1d225)

**Comment by sxyazi** [comment]
This could be a bug, would you like to create a bug report? https://github.com/sxyazi/yazi/issues/new?template=bug.yml

**Reply by giuskywalker** [reply]
https://github.com/sxyazi/yazi/issues/610

**Reply by sxyazi** [reply, marked as answer]
Resolved in that issue
